browser/extensions/brave_extension_provider.h

```diff
@@ -21,8 +21,7 @@ class BraveExtensionProvider : public ManagementPolicy::Provider {
                    base::string16* error) const override;
   bool MustRemainInstalled(const Extension* extension,
                            base::string16* error) const override;
-  static bool IsVetted(const extensions::Extension* extension);
-  static bool IsVetted(const std::string id);
+
```

Comment:
Nit: extra line?

Reply:
a blank line before `private:` is required by lint